pkg/prompts/prompts.go

```diff
@@ -23,23 +23,39 @@ import (
 const (
 	Yes = "Yes"
 	No  = "No"
+
+	Add      = "Add"
+	Del      = "Delete"
+	Preview  = "Preview"
+	MoreInfo = "More Info"
+	Done     = "Done"
+	Cancel   = "Cancel"
 )
 
 type Prompter interface {
 	CapturePositiveBigInt(promptStr string) (*big.Int, error)
-	CaptureAddress(promptStr string) (common.Address, error)
+	CaptureAddress(promptStr string, arg any) (any, error)
 	CaptureExistingFilepath(promptStr string) (string, error)
 	CaptureYesNo(promptStr string) (bool, error)
 	CaptureNoYes(promptStr string) (bool, error)
 	CaptureList(promptStr string, options []string) (string, error)
 	CaptureString(promptStr string) (string, error)
-	CaptureIndex(promptStr string, options []common.Address) (int, error)
+	CaptureIndex(promptStr string, options []interface{}) (int, error)
```

Comment:
probably can use `any` here also

Reply:
probably can be generic

Reply:
`any` is just an alias to `interface{}` but yes, I will change it here because I used `any` elsewhere. Why should I use generics, this works perfectly. Using generics is not necessarily better.

Reply:
Using generics you keep the most appropiate type for the domain of each function, and avoid boilerplate code to copy data.